Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -14,4 +14,10 @@
     <ProjectReference Include="..\Microsoft.Toolkit\Microsoft.Toolkit.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I'm guessing the issue has to do with the update of these packages here. Are these needed? 

Why dependency on .NET Standard 2? This will not be usable on Creators Update or bellow

Reply:
i can downgrade it to .net standard 1.4? those packages are not added by myself... i upgraded to .net standard 2.0.1 but the other packages were already in the solution.

so which version of .net standard shall i use?